tests/aws/services/lambda_/test_lambda_legacy.py

```diff
@@ -87,8 +89,8 @@ def test_logging_in_local_executor(create_lambda_function, handler_path, aws_cli
 
 
 @pytest.mark.skipif(not is_old_provider(), reason="test does not make valid assertions against AWS")
+@markers.aws.unknown
```

Comment:
This isn't enforced right now, but please keep the markers at a function level

Reply:
wasn't sure about that as well; will fix it 👍